### ISSUE TITLE
No module found FontProperties

### ISSUE THREAD

**Issue by wqh17101** [gventuri/pandas-ai#238]
### 🐛 Describe the bug

when the code generated by openai contain
```
from matplotlib.font_manager import FontProperties
```
It will crash for missing this import

**Comment by gventuri** [comment]
@wqh17101 thanks a lot for reporting. It could be that the library is not one of the whitelisted. Let me dig into it. In the menwhile, would you mind sharing the prompt and the output with "verbose=True"?

**Comment by jonbiemond** [comment]
> It could be that the library is not one of the whitelisted.

Correct, currently only `matplotlib.pyplot` is included in the environment. This exposes one of the problems with my implementation in #225. I guess we need to remove any imports of `matplotlib.pyplot` but persist any other `matplotlib.*` imports from the generated code.
